fiftyone/utils/eval/activitynet.py

```
@@ -368,8 +368,9 @@ def __init__(
         self.thresholds = (
             np.asarray(thresholds) if thresholds is not None else None
         )
+        self.full_classwise_AP = np.array(classwise_AP)
```

Pylint (4.0.2)

[convention] Attribute name "full_classwise_AP" doesn't conform to '[a-z_]([a-z0-9_]{0,30})$' pattern

(C0103)